benchmarks/naslib/problem.py

```diff
@@ -0,0 +1,143 @@
+import ast
+import os
+import sys
+from typing import Any
+from typing import Dict
+from typing import List
+from typing import Optional
+
+from kurobako import problem
+
+
+sys.stdout = open(os.devnull, "w")  # Suppress output
+
+
+import naslib.search_spaces as ss  # NOQA
+from naslib.search_spaces.core.graph import Graph  # NOQA
+from naslib.search_spaces.core.query_metrics import Metric  # NOQA
+import naslib.search_spaces.nasbench201.graph as nasbench201_graph  # NOQA
+from naslib.utils import get_dataset_api  # NOQA
+
+
+class NASLibProblemFactory(problem.ProblemFactory):
+    def __init__(self, search_space: Graph, **config: Any) -> None:
+        self._search_space = search_space
+        self._config = config
+
+    def specification(self) -> problem.ProblemSpec:
+        if isinstance(self._search_space, ss.NasBench201SearchSpace):
+            params = [
+                problem.Var(f"x{i}", problem.CategoricalRange(nasbench201_graph.OP_NAMES))
```

Comment:
Can we make the variable name more descriptive? In addition, `6` in the nextline seems to be a magic number. We may refer to the nasbench201 property or something.